Add sharing notifications feature documentation (#2876)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Core Concepts/workspaces-and-collaboration.md

* When you share with a SAML or LDAP group, all users who belong to that group will be able to access the asset
* If you decide to share with a team workspace, all the users on the workspace with the corresponding permissions will be able to access the asset.

When you share with a Workspace, all workspace members get access according to their role.

A workspace is a shared environment in which members of that workspace can access the same providers, boxes, and instances. Workspaces are contextual. When you switch workspaces you see a whole different group of instances, boxes, and providers, which belong only to that workspace.

In the sharing dialog of a resource, you can do the following:

* Add a new user or workspace to the sharing list
* Remove an existing user or workspace from the sharing list
* Change access permissions from an existing user or workspace

![Sharing dialog](../../images/cloud-application-manager/core-concepts/sharing-dialog.png)

The changes to be applied to the sharing list appear highlighted with a dotted border line, and will be effective once you click on the **Save** button.

### Sharing notifications

> The users that have been added to the sharing list of a resource, either a provider, box or instance, will see when accessing through the UI a notification badge consisting on a red dot that may include a number if several resources are available there. This badge will appear in several places to make him aware of the new resource he has access to:

Reviewer: ```suggestion
A notification badge consisting on a red dot will appear on the UI for those users users that have been added to the sharing list of a resource (i.e. provider, box or instance). The badge may include a number if two or more resources are available there. It will appear in several places to make the user aware of the new resource he has access to:
```

#### Clearing all notifications at once

There is also a new bell icon in the top navigation bar that provides an option to clear all notifications at once, so the user will get rid of all UI notifications that he currently has.

![Clear all notifications option](../../images/cloud-application-manager/core-concepts/sharing-clear-notifications.png)

### Sharing example

Say you have a Jenkins box that integrates and stages code for testing. You want to collaborate with other Jenkins experts to make the box configuration highly usable. So you give their workspace edit access. Next, the QA team needs this box to deploy and run tests, so you give their workspace view access. Now the QA team can deploy Jenkins instances, but as you’d expect, they aren’t allowed to change the underlying Jenkins box definition.

> **Notice** that providers and policy boxes cannot be shared with workspaces that are not in the same Cost Center. This avoids users outside of the Cost Center to deploy instances that will increase the cost associated with the Cost Center.

Reviewer: ```suggestion
**Notice** that providers and policy boxes cannot be shared with workspaces that are not in the same Cost Center. This avoids users outside one Cost Center to deploy instances that will increase the cost associated with another Cost Center.
```
